src/containerapp/azext_containerapp/_help.py

```diff
@@ -1548,13 +1548,13 @@
           az containerapp env java-component admin-for-spring update -g MyResourceGroup \\
               -n MyJavaComponentName \\
               --environment MyEnvironment \\
-              --configuration
+              --set-configurations
```

Comment:
Is this work with `--set-configuration`?

Reply:
Base on the `_params`, they are conflict:

> Add or update Java component configuration(s). Other existing configurations are not modified. Configurations must be in format \"<propertyName>=<value>\" \"<propertyName>=<value>\"..."

Reply:
The help message is wrong, --set-configurations could not delete configration, has updated to --remove-all-configurations